.codespellignore

```diff
@@ -0,0 +1,6 @@
+doubleclick
+thur
+neet
+bre
+wan
+keep-alives
```

Comment:
I kind of agree with this one, but I'm kind of torn as the normal spell checkers often flag it as a typo...